2.0/TextureEncodingTest/README.md

```diff
@@ -0,0 +1,23 @@
+# Texture Encoding Test
+
+## Screenshot
+
+![screenshot](screenshot/screenshot.png)
```

Comment:
Can we include a screenshot of what it looks like if the ICC profiles are not ignored?

Reply:
Custom gamma values and ICC profiles are more distinct now. Screenshot added.